operators/hack/gke-cluster.sh

```diff
@@ -37,23 +50,39 @@ create_cluster() {
         echo "-> GKE cluster is running."
         # make sure cluster config is exported for kubectl
         export_credentials
+	      # also make sure that vm.max_map_count is set if PSP is enabled
+        if [ "$PSP" == "1" ]; then
+            set_max_map_count
+        fi
         exit 0
     fi
 
+    local PSP_OPTION
+    if [ "$PSP" == "1" ]; then
+        PSP_OPTION="--enable-pod-security-policy"
+    fi
+
     echo "-> Creating GKE cluster..."
     gcloud beta container --project ${GCLOUD_PROJECT} clusters create ${GKE_CLUSTER_NAME} \
         --region "${GKE_CLUSTER_REGION}" --username "${GKE_ADMIN_USERNAME}" --cluster-version "${GKE_CLUSTER_VERSION}" \
         --machine-type "${GKE_MACHINE_TYPE}" --image-type "COS" --disk-type "pd-ssd" --disk-size "30" \
         --local-ssd-count "${GKE_LOCAL_SSD_COUNT}" --scopes "${GKE_GCP_SCOPES}" --num-nodes "${GKE_NODE_COUNT_PER_ZONE}" \
         --enable-cloud-logging --enable-cloud-monitoring --addons HorizontalPodAutoscaling,HttpLoadBalancing \
         --no-enable-autoupgrade --no-enable-autorepair --network "projects/${GCLOUD_PROJECT}/global/networks/default" \
-        --subnetwork "projects/${GCLOUD_PROJECT}/regions/${GKE_CLUSTER_REGION}/subnetworks/default"
+        --subnetwork "projects/${GCLOUD_PROJECT}/regions/${GKE_CLUSTER_REGION}/subnetworks/default" \
+        ${PSP_OPTION}
 
     # Export credentials for kubelet
     export_credentials
 
     # Create required role binding between the GCP account and the K8s cluster.
     kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$(gcloud auth list --filter=status:ACTIVE --format="value(account)")
+
+    # set vm.max_map_count if PSP is enabled
+    if [ "$PSP" == "1" ]; then
+        set_max_map_count
```

Comment:
this is done twice? (also at line 54)

Reply:
first one is when GKE already exist, same problem for `export_credentials`
It would need some refactoring, could be done in an other PR